src/gateway/kong-enterprise/index.md

```diff
@@ -4,89 +4,97 @@ badge: enterprise
 content_type: explanation
 ---
 
-In addition to the {{site.base_gateway}} open source, free, and plus features, users with an Enterprise plan also have access to Enterprise-only features. These features enhance the security, monitoring, and management capabilities of {{site.base_gateway}}.
+Kong Enterprise is the scalable, secure, and flexible API management solution that extends Kong Gateway, the fastest, most adopted API gateway, with enterprise plugins, developer portal, analytics, advanced security features, GUI's, and 24/7 support. It is the only solution that helps you accelerate your cloud journey by managing, securing, and monitoring connections between applications across hybrid and multi-cloud architectures, to help you scale faster and boost developer productivity. 
 
-The following sections describe key Kong Enterprise features. 
+## Enterprise Plugins
+
+{{site.base_gateway}} offers access to 400+ out-of-box enterprise and community plugins. {{site.base_gateway}} Enterprise offers exclusive versions of OSS plugins like the [Rate-Limiting Advanced plugin](hub/kong-inc/rate-limiting-advanced/) with added functionality such as the use of consumer groups, and database specific strategy. {{site.base_gateway}} Enterprise natively support gRPC and REST, WebSockets and integrate with Apollo GraphQL server and Apache Kafka services. These plugins can be leveraged to provide advanced connectivity features and solutions to {{site.base_gateway}} such as:
+
+* [Event gateways with Kafka](/hub/kong-inc/kafka-upstream/)
+* [GraphQL](/hub/kong-inc/graphql-proxy-cache-advanced/)
+* [Mocking](/hub/kong-inc/mocking/)
+* [Advanced data transformation](/hub/kong-inc/jq/)
+* [OPA Policy driven traffic management](/hub/kong-inc/opa/)
+* [API product tiers](/gateway/{{page.kong_version}}/admin-api/consumer-groups/reference/#main)
+
+[Get started plugins &rarr;](/hub/)
+
+## Dev Portal
+
+The Dev Portal provides a single source of truth for all developers to locate, access and consume APIs, similar to a traditional API Catalogue. [Dev Portal](/gateway/{{page.kong_version}}/kong-enterprise/dev-portal/) streamlines developer onboarding by offering a self-service developer experience to discover, register, and consume published services from the Service Hub catalog. This customizable experience can be used to match your own unique branding and highlights the documentation and interactive API specifications of your services. Enable application registration to automatically secure your APIs with a variety of authorization providers.
```

Comment:
Here (and in the other sections), I think you can remove the links to the overview page for the feature (ex. Dev Portal overview) in the paragraph and just leave the "Learn more about...." link at the bottom. Since they both link to the same page.

Reply:
done